src/components/navigation/Navigation.jsx

```diff
@@ -1,40 +1,52 @@
 import bcgovlogo from 'assets/images/bcgov-h.png';
+import slamlogo from 'assets/images/slam-logo.png';
 import Image from 'next/image';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
-import React from 'react';
+import {React, useState } from 'react';
+import { AvatarChip, UserMenu } from 'components';
+
+
+
```

Comment:
I'm a stickler for un-needed white space. What's best practice between imports and first line of code? 2 or 3?

Reply:
I usually leave one whitespace line